### PR TITLE
add permissions to github_token

### PR DESCRIPTION
**Pull request by RoiTabach**

### CODE REVIEW

**Comment by elongl** [comment]
I believe the [`permissions`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions) key needs to be defined on either the workflow globally, or a specific job, not a step.
image
Thanks for noticing!

File: .github/workflows/remind-docs-and-tests.yml [hunk 1]
```diff
@@ -9,8 +9,9 @@ jobs:
       - uses: wow-actions/auto-comment@v1
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            permissions: read-all|write-all
```
Comment (RoiTabach) [inline .github/workflows/remind-docs-and-tests.yml:12]: @elongl you mean something like 

```suggestion
      permissions: read-all|write-all
      - uses: wow-actions/auto-comment@v1
        with:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

File: .github/workflows/remind-docs-and-tests.yml [hunk 2]
```diff
@@ -6,11 +6,12 @@ jobs:
   run:
     runs-on: ubuntu-latest
     steps:
+      permissions: read-all|write-all
```
Comment (elongl) [inline .github/workflows/remind-docs-and-tests.yml:9]: This is still under `steps`.
It needs to be placed right after the job's name.

**Comment by elongl** [comment]
image
It also says so on the action running for this PR so you can tell if it works or not.

**Pull request closed without merging**